docs/index.html

```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
```

Comment:
Can we serve this page in flask? It would be really useful to have `https://cachito.example.com/docs` serving this file to provide the accurate documentation for a given deployment.